### DISCUSSION TITLE
Huge bodies of water, instead of land

### DISCUSSION THREAD

**Discussion by VictorV0000** [kubilus1/autoortho#475] [General]
So i'm curreently doing a flight for ATL-JAC and came across this hugve body of water that shouldn't be there. idk how big it is. And this is with AutoOrtho.
![757-200_xp12 - 2024-01-07 15 59 43](https://github.com/kubilus1/autoortho/assets/132933933/002bb37a-ccc4-4a50-99f9-6812ef015b48)

**Comment by filipto861** [comment]
I have exactly the same issue. Has anyone found a solution yet?
